cases/v41/const.py

```diff
@@ -41,91 +28,53 @@
 ]
 dashboard_nodectl_id = [18545, 18547, 18550, 18551, 18552, 18830, 18831]
 
-
-# test_dashboard_ui
 dashboard_ui = [
     'check_node_status_func', 'check_node_health_func', 'check_node_info_func',
     'check_network_func', 'check_system_log_func', 'check_storage_func',
-    'check_ssh_key_func'
-]
-dashboard_ui_id = [18534, 18535, 18536, 18540, 18541, 18542, 18543]
-
-
-# test_dashboard_ui_efi
-dashboard_ui_efi = [
-    'check_nodestatus_efi'
+    'check_ssh_key_func', 'check_nodestatus_efi', 'check_nodestatus_fc'
 ]
-dashboard_ui_efi_id = [18539]
+dashboard_ui_id = [18534, 18535, 18536, 18540, 18541, 18542, 18543, 18539, 18538]
```

Comment:
dashboard_ui_efi and dashboard_ui_fc are just for efi machine and fc machine test, It's better to leave them alone.

Reply:
changes are not from me. Not sure how they appeared here